src/main/java/org/folio/des/service/JobExecutionService.java

```diff
@@ -47,6 +55,30 @@ public JobCommand prepareStartJobCommand(Job job) {
     return jobCommand;
   }
 
+  public JobCommand prepareResendJobCommand(Job job) {
+    validateIncomingExportConfig(job);
+    JobCommand jobCommand = buildResendJobCommand(job);
+
+    Map<String, JobParameter> params = new HashMap<>();
+    String exportConfigId = job.getExportTypeSpecificParameters()
+      .getVendorEdiOrdersExportConfig()
+      .getExportConfigId().toString();
+
+    ModelConfiguration modelConfiguration = manager.getConfigById(exportConfigId);
+    ExportConfig config = defaultModelConfigToExportConfigConverter.convert(modelConfiguration);
+    Optional.ofNullable(config.getExportTypeSpecificParameters()).
+      map(ExportTypeSpecificParameters::getVendorEdiOrdersExportConfig)
+      .ifPresent(vendorEdiOrdersExportConfig ->
+        params.put("edifactOrdersExport",
```

Comment:
Lets move these param names to constants

Reply:
fixed